test/should_pass_SUITE.erl

```diff
@@ -0,0 +1,107 @@
+
+end_per_testcase(_TestCase, _Config) ->
+    ok.
+
+all() ->
+    [{group, all_tests}].
+
+groups() ->
+    [{all_tests, [parallel], all_tests()}].
+
+all_tests() ->
+    [alias_in_pattern,andalso_any,ann_types,annotated_types,any,
+     any_doesnt_have_type_none_pass,any_pattern,bc_pass,
+     binary_exhaustiveness_checking,binary_in_union,binary_literal_pattern,
+     bitstring,block_scope,bool,bounded_funs,
+     call_intersection_function_with_union_arg_pass,'case',
+     case_of_record_with_user_defined,catch_expr_pass,covariant_map_keys_pass,
+     cyclic_otp_specs,erlang_error_args_none_pass,exhaustiveness_union_types,
+     factorial,float,flow,fun_capture,fun_spec,guard,guard_sequences_pass,if_expr,
+     imported,int,intersection_pass,intersection_with_any_pass,iodata,issue131,lc,
+     lc_generator_not_none,lc_var_binds_in_filters,list,list_concat_op_pass,
+     list_exhaustiveness_checking_regressions,
+     list_exhaustiveness_checking_regressions2,
+     list_exhaustiveness_checking_unreachable_clause_regression,list_infer_pass,
+     list_op_pass,listsspecs,map,map_as_argument_update,map_creation,
+     map_field_valid_update,map_infer_pass,map_passing_expr,map_passing_subtyping,
+     map_pattern,map_refinement,map_update,map_update_with_record_field,
+     messaging_pass,minimised_gradualizer_fmt,minus,module_info_higher_arity,
+     module_info_pass,named_fun_infer_pass,named_fun_pass,negate_none,
+     nested_pattern_match,non_neg_plus_pos_is_pos_pass,nonempty_cons,
+     nonempty_list_match_in_head_exhaustive,nonempty_string,
+     nonexhaustive_record_pattern,opaque,operator_pattern_pass,operator_subtypes,
+     other_module,pattern_bind_reuse,pattern_record,pattern_with_ty_vars,
+     poly_lists_map_constraints_pass,poly_lists_map_pass,poly_map_pattern,
+     poly_pass,poly_pass_infer,poly_pass_no_solve_constraints,
+     poly_union_lower_bound_pass,preludes,qlc_test,record_info,record_refinement,
+     record_union_pass,record_union_with_any_should_pass,record_var,
+     record_wildcard_pass,record_with_user_defined,records,
+     recursive_call_with_remote_union_return_type_pass,recursive_types_passing,
+     refine_comparison,refine_mismatch_using_guard_bifs,remote_types,
+     remote_types_pass,return_fun,rigid_type_variables,rigid_type_variables_pass,
+     scope,send_pass,sets_set,shortcut_ops_pass,
+     spec_and_fun_clause_intersection_pass,stuff_as_top,'try',try_expr,tuple,
+     tuple_union_pass,tuple_union_pat,tuple_union_pattern_pass,type_decl,
+     type_pattern,type_refinement_pass,type_variable,type_vars_term,
+     typed_record_field_access,unary_negate_union_with_user_type_pass,unary_plus,
+     underscore,user_type_in_pattern_body,user_types,var,var_fun,varbind_in_block,
+     varbind_in_case,varbind_in_function_head,varbind_in_lc,variable_binding,
+     variable_binding_leaks].
+
+should_pass_template(_@File) ->
+    ?assertEqual(ok, gradualizer:type_check_file(_@File, [{form_check_timeout_ms, 2000}])).
```

Comment:
The EUnit tests print a Gradualizer's error message describing the type error when a test fails. I find it useful for quickly glancing over what's going on without having to inspect all the failed test files one by one. Do you think it's possible with CT?

Reply:
It's available in the CT HTML report, but that requires a little bit of clicking in the browser. It might be possible to get in the shell, too, but I'd have to think some more about it.

Reply:
IIRC, CT supports plain text reports too. I don't know it well, but if it's too complicated, maybe it isn's worth it.

If we run it ourselves instead of relying on rebar3, we'll have better control over it.